yetanotherco/aligned_layer
# feat: aggregator total gas paid metric

# Aggregator total gas paid metric

## Description

This PR adds aggregator total gas paid while responding to tasks metric to the grafana dashboard.

## How to test

1. Start all services locally, including the metrics
2. Start sending proofs
3. Go to the "System Data" dashboard, you should see a new panel showing the total amount of ETH paid by the aggregator, right down the gas price of the last batch.
4. Shut down the aggregator and start it over again
5. You should see that on new batch finished, the metric will keep accumulating on previous data.

## Type of change

Please delete options that are not relevant.


## Checklist

  - [Link to the open issue addressing it]()
